runhouse/resources/hardware/cluster.py

```diff
@@ -591,6 +594,10 @@ def _use_custom_cert(self):
     def _use_custom_key(self):
         return Path(self.cert_config.key_path).exists()
 
+    @property
+    def _use_local_telemetry(self) -> bool:
```

Comment:
can remove this property getter since its just a boolean?

Reply:
Done.